compiler/src/typed/parmatch.re

```diff
@@ -268,6 +275,7 @@ let const_compare = (x, y) =>
       Const_wasmi64(_) |
       Const_wasmf32(_) |
       Const_wasmf64(_) |
+      Const_bigint(_) | // <- impossible
```

Comment:
Mind adding an explanation (or dropping the comment if this would be correct if it weren't impossible)?

Reply:
Yeah, I don't think this is impossible, after all, but I do think that this is the correct behavior. I've deleted the comment.